packages/block-editor/src/components/block-controls/fill.js

```diff
@@ -38,7 +33,10 @@ export default function BlockControlsFill( {
 					// Children passed to BlockControlsFill will not have access to any
 					// React Context whose Provider is part of the BlockControlsSlot tree.
 					// So we re-create the Provider in this subtree.
-					const value = ! isEmpty( fillProps ) ? fillProps : null;
+					const value =
+						fillProps && Object.keys( fillProps ).length > 0
```

Comment:
`fillProps` should be an object since it's coming from component props:

https://github.com/WordPress/gutenberg/blob/b4887bfaf366425605f40c0b60b03d933b12d21b/packages/components/src/slot-fill/index.js#L49